sklearn/metrics/_plot/det_curve.py

```diff
@@ -300,6 +301,13 @@ def plot(self, ax=None, *, name=None, **kwargs):
         line_kwargs = {} if name is None else {"label": name}
         line_kwargs.update(**kwargs)
 
+        # avoid inf from sp.stats.norm.ppf
+        eps = 1e-8
```

Comment:
This is a bit arbitrary. Why this value. Does the choice has an impact for some use cases?

Reply:
Yes, it is arbitrary but I just set it to be smaller than `atol` in the tests. The impact is plotting, as matplotlib does not natively deal with `inf`. Should I add such explanation to the comment?

Reply:
What would be the impact of using `np.finfo(fpr.dtype).eps` here?